sdk/tables/azure-data-tables/README.md

```diff
@@ -267,33 +293,81 @@ service_client.get_service_stats(logging_enable=True)
+<!-- Still needed -->
+[Tables_pypi]: https://pypi.org/project/azure-data-tables
+[Tables_ref_docs]: https://aka.ms/azsdk/python/tables/docs
+[Tables_product_doc]: https://docs.microsoft.com/azure/storage/
+[Tables_samples]: https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/tables/azure-data-tables/samples
+
+[azure_subscription]: https://azure.microsoft.com/free/
+[azure_storage_account]: https://azure.microsoft.com/free/
+[azure_cosmos_account]: https://docs.microsoft.com/azure/cosmos-db/account-overview
+[pip_link]: https://pypi.org/project/pip/
+
+[azure_create_cosmos]: https://docs.microsoft.com/azure/cosmos-db/create-cosmosdb-resources-portal
+[azure_portal_create_account]: https://docs.microsoft.com/azure/storage/common/storage-quickstart-create-account?tabs=azure-portal
+[azure_powershell_create_account]: https://docs.microsoft.com/azure/storage/common/storage-quickstart-create-account?tabs=azure-powershell
+[azure_cli_create_account]: https://docs.microsoft.com/azure/storage/common/storage-quickstart-create-account?tabs=azure-cli
+
+[azure_cli_account_url]:https://docs.microsoft.com/cli/azure/storage/account?view=azure-cli-latest#az-storage-account-show
+[azure_powershell_account_url]:https://docs.microsoft.com/powershell/module/az.storage/get-azstorageaccount
+[azure_portal_account_url]:https://docs.microsoft.com/azure/storage/common/storage-account-overview#storage-account-endpoints
+
+[azure_authorization]: https://docs.microsoft.com/azure/storage/common/storage-auth
+[azure_sas_token]: https://docs.microsoft.com/azure/storage/common/storage-sas-overview
+[azure_shared_key]: https://docs.microsoft.com/rest/api/storageservices/authenticate-with-shared-key/
+
+<!-- Still needed -->
+[table_service_client_docs]: https://aka.ms/azsdk/python/tables/docs
+<!-- Still needed -->
+[table_client_docs]: https://aka.ms/azsdk/python/tables/docs
+
+[azure_core_ref_docs]: https://azuresdkdocs.blob.core.windows.net/$web/python/azure-core/latest/azure.core.html
+[azure_core_readme]: https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/core/azure-core/README.md
+
+[python_logging]: https://docs.python.org/3/library/logging.html
+[tables_error_codes]: https://docs.microsoft.com/rest/api/storageservices/table-service-error-codes
+
+<!-- Still needed -->
+[samples_authentication]: https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/tables/azure-data-tables/samples/table_samples_authentication.py
+<!-- Still needed -->
+[samples_services]: https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/tables/azure-data-tables/samples/table_samples_authentication.py
+<!-- Still needed -->
+[samples_client]: https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/tables/azure-data-tables/samples/table_samples_client.py
+
+[msft_oss_coc]:https://opensource.microsoft.com/codeofconduct/
+[msft_oss_coc_faq]:https://opensource.microsoft.com/codeofconduct/faq/
+[contact_msft_oss]:mailto:opencode@microsoft.com
```

Comment:
add tracking pixel - example here (last line - just change the path) https://raw.githubusercontent.com/Azure/azure-sdk-for-python/master/sdk/eventhub/azure-eventhub/README.md

Reply:
Can you verify I have this one correct?

Reply:
I don't see it here, but it should be : 
![Impressions](https://azure-sdk-impressions.azurewebsites.net/api/impressions/azure-sdk-for-python/sdk/tables/azure-data-tables/README.png)

Reply:
Sorry, think I overwrote it when tackling merge conflicts